cw_decred/lib/wallet.dart

```diff
@@ -380,7 +380,7 @@ abstract class DecredWalletBase
         totalAmt = totalIn;
       } else if (out.cryptoAmount != null) {
         final coins = double.parse(out.cryptoAmount!);
-        amt = (coins * 1e8).toInt();
```

Comment:
This doesn't seem to be as accurate so changed to `round`
The following will print 29999
```
    final coins = .0003;
    final thing = (coins * 1e8).toInt();
    print("${thing}");
```